prow/config.yaml

```diff
@@ -151,6 +151,11 @@ presubmits:
     rerun_command: "@k8s-bot pull-kubernetes-e2e-gce test this"
     trigger: "@k8s-bot pull-kubernetes-e2e-gce test this"
 
+  - name: pull-kubernetes-e2e-gce-bazel
+    context: pull-kubernetes-e2e-gce-bazel
+    rerun_command: "@k8s-bot pull-kubernetes-e2e-gce-bazel test this"
+    trigger: "@k8s-bot pull-kubernetes-e2e-gce-bazel test this"
```

Comment:
assume you want to run it as a prow pod rather than a jenkins job? Then you'll need to fill up the pod spec here.